custom_components/dreo/climate.py

```diff
@@ -0,0 +1,350 @@
+"""Support climate control for some Dreo devices"""
+# Suppress warnings about DataClass constructors
+# pylint: disable=E1123
+
+# Suppress warnings about unused function arguments
+# pylint: disable=W0613
+from __future__ import annotations
+
+from typing import Any
+from dataclasses import dataclass
+import logging
+
+from .haimports import * # pylint: disable=W0401,W0614
+from .basedevice import DreoBaseDeviceHA
+from .pydreo import PyDreo
+from .pydreo.pydreobasedevice import PyDreoBaseDevice
+from .pydreo import (
+    PyDreoHeater,
+    HEATER_MODE_OFF,
+    HEATER_MODE_COOLAIR,
+    HEATER_MODE_HOTAIR,
+    HEATER_MODE_ECO,
+    MODE_LEVEL_MAP,
+    LEVEL_MODE_MAP,
+    HEAT_RANGE,
+    ECOLEVEL_RANGE
+)
+
+from homeassistant.components.climate import (
```

Comment:
Please consider moving these into .haimports.  I had them all there to avoid some collisions AFAIR

Reply:
Will do.

Reply:
Done.